catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSource.java

```diff
@@ -722,8 +726,16 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
 
     try {
       LOGGER.debug("WFS Source {}: Getting hits.", getId());
-      final WfsFeatureCollection hitsResponse = wfs.getFeature(getHits);
-      final long totalHits = hitsResponse.getNumberOfFeatures();
+
+      long totalHits;
+
+      try {
+        final WfsFeatureCollection hitsResponse = wfs.getFeature(getHits);
+        totalHits = hitsResponse.getNumberOfFeatures();
+      } catch (ResponseProcessingException e) {
+        LOGGER.warn("WFS Source {} hit count query returned an exception", getId(), e);
```

Comment:
❓ Can we also get this exception into the processing details of the response?

Reply:
oops, meant to do that. I'll fix it.

Reply:
I'm not sure if I should put in just the top-level exception message or the full stack trace. In this case, the top-level exception message doesn't say anything useful, since the real exception is nested two levels deeper. But the full stack trace is pretty wordy. I'm going to go with the full stack trace, but let me know if you want something different.

Reply:
Full stack trace is what I was thinking. That is what we usually do for processing details.